### ISSUE TITLE
Cypress doesn't correctly handle the -s option

### ISSUE THREAD

**Issue by shauvik** [cypress-io/cypress#120]
Cypress seems to be looking inside it's own app for the spec to test..

```
$ ls cypress/integration/end-to-end/setup.js 
cypress/integration/end-to-end/setup.js

$ cypress run -s _does_not_exit_/end-to-end/setup.js 
Can't find test spec: /Users/shauvik/src/mn/projects/fullstory/cypress/integration/_does_not_exit_/end-to-end/setup.js

$ cypress run -s end-to-end/setup.js 

Tests should begin momentarily...

Error: ENOENT: no such file or directory, stat '/Applications/Cypress.app/Contents/Resources/app/end-to-end/setup.js'
  at Error (native)
  at fs.statSync (fs.js:897:18)
  at Object.fs.statSync (ATOM_SHELL_ASAR.js:264:16)
  at Snockets.module.exports.Snockets.stat (/Applications/Cypress.app/Contents/Resources/app/node_modules/snockets/lib/snockets.js:367:22)
  at Snockets.module.exports.Snockets.readFile (/Applications/Cypress.app/Contents/Resources/app/node_modules/snockets/lib/snockets.js:377:19)
  at Snockets.module.exports.Snockets.updateDirectives (/Applications/Cypress.app/Contents/Resources/app/node_modules/snockets/lib/snockets.js:280:19)
  at Snockets.module.exports.Snockets.getCompiledChain (/Applications/Cypress.app/Contents/Resources/app/node_modules/snockets/lib/snockets.js:76:19)
  at Object.module.exports.handle (/Applications/Cypress.app/Contents/Resources/app/index.js:1179:31)
  at /Applications/Cypress.app/Contents/Resources/app/index.js:3908:19
  at Layer.handle [as handle_request] (/Applications/Cypress.app/Contents/Resources/app/node_modules/express/lib/router/layer.js:82:5)
  at next (/Applications/Cypress.app/Contents/Resources/app/node_modules/express/lib/router/route.js:110:13)
  at Route.dispatch (/Applications/Cypress.app/Contents/Resources/app/node_modules/express/lib/router/route.js:91:3)
  at Layer.handle [as handle_request] (/Applications/Cypress.app/Contents/Resources/app/node_modules/express/lib/router/layer.js:82:5)
  at /Applications/Cypress.app/Contents/Resources/app/node_modules/express/lib/router/index.js:267:22
  at Function.proto.process_params (/Applications/Cypress.app/Contents/Resources/app/node_modules/express/lib/router/index.js:321:12)
  at next (/Applications/Cypress.app/Contents/Resources/app/node_modules/express/lib/router/index.js:261:10)
  at Layer.handle [as handle_request] (/Applications/Cypress.app/Contents/Resources/app/node_modules/express/lib/router/layer.js:78:12)
  at trim_prefix (/Applications/Cypress.app/Contents/Resources/app/node_modules/express/lib/router/index.js:302:13)
  at /Applications/Cypress.app/Contents/Resources/app/node_modules/express/lib/router/index.js:270:7
  at Function.proto.process_params (/Applications/Cypress.app/Contents/Resources/app/node_modules/express/lib/router/index.js:321:12)
  at next (/Applications/Cypress.app/Contents/Resources/app/node_modules/express/lib/router/index.js:261:10)
  at compression (/Applications/Cypress.app/Contents/Resources/app/node_modules/compression/index.js:217:5)
  at Layer.handle [as handle_request] (/Applications/Cypress.app/Contents/Resources/app/node_modules/express/lib/router/layer.js:82:5)
  at trim_prefix (/Applications/Cypress.app/Contents/Resources/app/node_modules/express/lib/router/index.js:302:13)
  at /Applications/Cypress.app/Contents/Resources/app/node_modules/express/lib/router/index.js:270:7
  at Function.proto.process_params (/Applications/Cypress.app/Contents/Resources/app/node_modules/express/lib/router/index.js:321:12)
  at next (/Applications/Cypress.app/Contents/Resources/app/node_modules/express/lib/router/index.js:261:10)
  at cookieParser (/Applications/Cypress.app/Contents/Resources/app/node_modules/cookie-parser/index.js:56:14)
  at Layer.handle [as handle_request] (/Applications/Cypress.app/Contents/Resources/app/node_modules/express/lib/router/layer.js:82:5)
  at trim_prefix (/Applications/Cypress.app/Contents/Resources/app/node_modules/express/lib/router/index.js:302:13)
  at /
```

**Comment by shauvik** [comment]
It would be great if cypress also supported specifying a directory of specs like:

`cypress run --spec end-to-end`
OR 
`cypress run --spec end-to-end/*.js`

Not sure if this was already in the plan.

**Comment by shauvik** [comment]
Just found that `cypress ci` doesn't support the `--spec/-s` option. It would be great if it were to support that too.

**Comment by brian-mann** [comment]
Fixed in [`0.15.3`](https://github.com/cypress-io/cypress/wiki/Changelog#0153-04102016).

**Comment by brian-mann** [comment]
Open a new issue for passing a dir to `--spec` since that is a new feature, and we'll track it better that way.

As for passing `--spec` to `cypress ci` that's going to be a no go. I understand why you want to do it since you're essentially using `cypress ci` as an alternate to `cypress run` due to not being able to log in from the command line.

At the moment there's not much difference between `cypress ci` and `cypress run` but soon they will be two very different things. One will spin up real browser instances, record the screen, take screenshots, do analysis, and make the build results available to you, which will take up build hours. The other will simply run the tests locally and will not record any permanent results.

This platform will be rolling out fairly soon and if we just added the ability to log in from the command line then my guess is you'd use `cypress run`.
